docs/attribution/0001-analytics.rst

```diff
@@ -5,15 +5,11 @@
 .. _analytics:
 
 =================
-Website analytics
+Mozorg analytics
 =================
 
-***********
-Mozorg Mode
-***********
-
 Google Tag Manager (GTM)
-------------------------
+************************
```

Comment:
**Question**: Why the change from the previous symbol (`-`) to `*`

Reply:
To change the heading levels. Everything got bumped up a level when I removed the Pocket stuff.